images/pegasus/scripts/startup.sh

```diff
@@ -0,0 +1,136 @@
+fi
+if test -f /bioses/prod.keys; then
+    gow_log "YUZU - /bioses/title.keys present, copy to YUZU folder"
+	  mkdir -p $YUZU_CFG_DIR/keys/
+    cp -u $HOME/bioses/title.keys $YUZU_CFG_DIR/keys/title.keys
+fi
+
+#########################################
+# Configure CEMU
+#########################################
+CEMU_CFG_DIR=$HOME/.config/Cemu
+
+gow_log "CEMU - Setting default sound device and run in fullscreen, if not edited"
+cp -u /cfg/cemu/settings.xml /cfg/cemu/settings_new.xml
+searchString="<TVDevice>replace_me</TVDevice>"
+replaceString="<TVDevice>${PULSE_SINK}</TVDevice>"
+sed -i -e "s|$searchString|$replaceString|g" /cfg/cemu/settings_new.xml
+searchString="<PadDevice>replace_me</PadDevice>"
+replaceString="<PadDevice>${PULSE_SINK}</PadDevice>"
+sed -i -e "s|$searchString|$replaceString|g" /cfg/cemu/settings_new.xml
+# Set the date of the file back to the original date
+touch -r /cfg/cemu/settings.xml /cfg/cemu/settings_new.xml
+cp -u /cfg/cemu/settings_new.xml $CEMU_CFG_DIR/settings.xml
+
+#########################################
+# Configure RPCS3
+#########################################
+RPCS3_CFG_DIR=$HOME/.config/rpcs3
+
+gow_log "RPCS3 - Copying custom config (disable Auto-Update pop-up), if not edited"
+mkdir -p $RPCS3_CFG_DIR/GuiConfigs/
+cp -u /cfg/rpcs3/CurrentSettings.ini $RPCS3_CFG_DIR/GuiConfigs/CurrentSettings.ini
+
+gow_log "RPCS3 - Setting default sound device and run in fullscreen, if not edited"
+mkdir -p $RPCS3_CFG_DIR/
+cp -u /cfg/rpcs3/config.yml /cfg/rpcs3/config_new.yml
+searchString="Audio Device: \"@@@default@@@\""
+replaceString="Audio Device: \"${PULSE_SINK}\""
+sed -i -e "s|$searchString|$replaceString|g" /cfg/rpcs3/config_new.yml
+# Set the date of the file back to the original date
+touch -r /cfg/rpcs3/config.yml /cfg/rpcs3/config_new.yml
+cp -u /cfg/rpcs3/config_new.yml $RPCS3_CFG_DIR/config.yml
+
+#########################################
+# Configure XEMU
+#########################################
+XEMU_CFG_DIR=$HOME/.local/share/xemu
+gow_log "XEMU - Copying custom config - settings, if not edited"
+mkdir -p $XEMU_CFG_DIR/xemu/
+cp -u /cfg/xemu/xemu.toml $XEMU_CFG_DIR/xemu/xemu.toml
+gow_log "XEMU - Copying basic hard drive (/bioses/xbox_hdd.gcow2), if present and not edited"
+if [ -f "/bioses/xbox_hdd.qcow2" ]; then
+    cp -u /bioses/xbox_hdd.qcow2 $XEMU_CFG_DIR/xemu/xbox_hdd.qcow2
+fi
+
+if [ -n "$RUN_GAMESCOPE" ]; then
+  echo "Gamescope - Starting Pegasus"
+  export GAMESCOPE_WIDTH=${GAMESCOPE_WIDTH:-1920}
+  export GAMESCOPE_HEIGHT=${GAMESCOPE_HEIGHT:-1080}
+  export GAMESCOPE_REFRESH=${GAMESCOPE_REFRESH:-60}
+  export GAMESCOPE_MODE=${GAMESCOPE_MODE:-"-b"}
+  /usr/games/gamescope "${GAMESCOPE_MODE}" -W "${GAMESCOPE_WIDTH}" -H "${GAMESCOPE_HEIGHT}" -r "${GAMESCOPE_REFRESH}" -- pegasus-fe
+else
+  echo "Sway - Starting Pegasus"
+  export XDG_SESSION_TYPE=wayland
+  mkdir -p $HOME/.config/sway/
+  cp /cfg/sway/config $HOME/.config/sway/config
+  exec sway --unsupported-gpu
+fi
```

Comment:
This should be moved in a separate script like `launch-comp.sh` possibly in the `base-app` image and just referenced in all images; we shouldn't copy paste it around like this.

Reply:
Done!